src/Semantic/Api/Symbols.hs

```diff
@@ -108,19 +105,17 @@ symbolsToSummarize = ["Function", "Method", "Class", "Module", "Call", "Send"]
 class ToTags t where
   tags :: Language -> [Text] -> Source -> t Loc -> [Tag]
 
-instance IsTaggable syntax => ToTags (Term syntax) where
-  tags = runTagging
+instance (Parser.TermMode term ~ strategy, ToTagsBy strategy term) => ToTags term where
+  tags = tagsBy @strategy
```

Comment:
Some advanced overlap here to select between the two approaches and avoid having to define per-language instances.